src/applications/yellow-ribbon/containers/SearchForm/index.unit.spec.jsx

```diff
@@ -16,7 +16,7 @@ describe('Yellow Ribbon container <SearchForm>', () => {
     // open additional info tip
     tree.find('va-additional-info').simulate('click');
     expect(select.length).to.be.equal(1);
-    expect(input.length).to.be.equal(4);
+    expect(input.length).to.be.equal(2);
```

Comment:
There are only 2 inputs here so I'm not sure why this was checking for 4. Test was failing otherwise.

![Screenshot 2023-09-27 at 3 31 35 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/cdfee4e6-87bb-4c7b-ae5b-6f7d17aef915)